website/cue/reference/components/sinks/splunk_hec.cue

```diff
@@ -3,9 +3,6 @@ package metadata
 components: sinks: _splunk_hec: {
 	configuration: {
 		acknowledgements: {
-			common:      false
-			description: "The configuration for Splunk HEC indexer acknowledgement client behavior."
-			required:    false
```

Comment:
Why is this going away?

Reply:
It conflicts with/shadows the global `acknowledgements` options, and that breaks validation.